src/components/color-picker/interfaces.ts

```diff
@@ -1,7 +1,5 @@
 import type Color from "color";
 
-export type ColorAppearance = "default" | "minimal" | "solid";
```

Comment:
I couldn't find any references to this "default" value.

Reply:
`default` is deprecated [here](https://github.com/Esri/calcite-components/pull/4744). I think its safe to remove it.